src/test/java/com/elasticbox/jenkins/tests/PullRequestManagerTest.java

```diff
@@ -0,0 +1,255 @@
+
+        LOGGER.fine("launching one pull request that triggers job 1\n");
+        pullRequest.open();
+        Thread.sleep(10000);
+
+        LOGGER.fine("Check that the job 0 is triggered");
+        waitForBuildTriggered(project, NEXTBUILD_TIMEOUT_LONG, "testPullRequestsForMultipleProjects");
+
+        Assert.assertNotNull(MessageFormat.format("Build is not triggered on opening of pull request {0} after {1} minutes",
+                pullRequest.getGHPullRequest().getHtmlUrl(), TimeUnit.SECONDS.toMinutes(NEXTBUILD_TIMEOUT_LONG)),
+                project.getLastBuild() );
+
+        FreeStyleBuild projectLastBuild1_0 = project.getLastBuild();
+        LOGGER.fine("The job 0 have triggered build with number " + projectLastBuild1_0.number);
+
+
+        LOGGER.fine("Creating new job 2 listening trigger from the same repository...\n");
+        FreeStyleProject project2 = createJenkinsJob2();
+
+        testAddingProjectToGHRepository(project2);
+
+        testSamePullRequestsTriggersMultipleProjects(project2);
+
+    }
+
+    public void testAddingProjectToGHRepository(FreeStyleProject project2) throws Exception {
+
+        LOGGER.fine("test ---- testAddingProjectToGHRepository\n");
+
+        // Check that job 2 has no builds
+        waitForBuildTriggered(project2, NEXTBUILD_TIMEOUT, "testAddingProjectToGHRepository 0");
+
+        Assert.assertNull(MessageFormat.format("Build in job 2 should not be triggered yet, on opening of pull request {0} after {1} minutes",
+                pullRequest.getGHPullRequest().getHtmlUrl(), TimeUnit.SECONDS.toMinutes(NEXTBUILD_TIMEOUT)),
+                project2.getLastBuild() );
+
+        // Wait till existing trigger Phrase is built, then send new trigger comment to both projects
+
+        final List<JSONObject> instances = new ArrayList<JSONObject>();
+
+        int count = 0;
+        LOGGER.fine("Testing trigger Phrase...");
+        final String triggerPhrase = TEST_TRIGGER_PHRASE;
+        pullRequest.comment(triggerPhrase + (count++) );
+        Assert.assertNull(MessageFormat.format("Unexpected build triggered with comment ''{0}''",
+                triggerPhrase),
+                waitForNextBuild(project, NEXTBUILD_TIMEOUT, "Waiting build triggered with comment " + triggerPhrase));
+
+        updateTriggerPhrase(project, triggerPhrase);
+        updateTriggerPhrase(project2, triggerPhrase);
+        pullRequest.comment(triggerPhrase + (count++) );
+        // Next build trigger is the same for both projects. Testing one of them is enough
+        waitForNextBuild(project, NEXTBUILD_TIMEOUT, "ensureBuildTriggeredInProject");
+
+        // Test both projects building this pullRequest and this commit
+
+        LOGGER.fine("Check that the job 1 build is started");
+        waitForBuildTriggered(project, NEXTBUILD_TIMEOUT_LONG, "testAddingProjectToGHRepository 1");
+
+        FreeStyleBuild projectLastBuild1 = project.getLastBuild();
+        if (projectLastBuild1 == null) {
```

Comment:
What does this log accomplish? If the last build is null the assertNotNull will trigger.